### DISCUSSION TITLE
Running Sync after updates to options.lua

### DISCUSSION THREAD

**Discussion by cpr-troyvit** [LazyVim/LazyVim#4349] [Q&A] [answered]
I'm loving LazyVim so far, but I just failed in my first sync. I see this error:

    ● LazyVim 8.17ms  start
        You have local changes in `/home/tvitullo/.local/share/nvim/lazy/LazyVim`:
          * lua/lazyvim/config/options.lua
          * lua/lazyvim/plugins/editor.lua
        Please remove them to update.
        You can also press `x` to remove the plugin and then `I` to install it again.

I definitely don't want to remove the plugin and install it again, but I'm also pretty sure I want to keep my changes in options.lua and editor.lua. Besides -- I don't know what the differences are regardless.

What's the normal thing to do in this instance? My understanding is that LazyVim is highly opinionated and is designed for devs to make tweaks to fit their own flow, but how can I keep those tweaks from breaking updates?

**Comment by dpetka2001** [comment, marked as answer]
The error says clearly what's wrong. You never make changes in `~/.local/share/nvim/lazy/LazyVim` directory. You only put your custom stuff in `~/.config/nvim/lua/config/` or `~/.config/nvim/lua/plugins/` directory. Kindly read the docs for what you have to do.

**Comment by cpr-troyvit** [comment]
Thank you for your kind and generous help. While the documentation has so far failed to help me, your advice and your positive attitude have been like a beacon of hope (especially the positive attitude!). As you say, the error does describe what's wrong, it just doesn't say what to do about it. The documentation offers advice on how to fix the issue, but that advice fails. I'll be able to figure it out from here though thanks (once again) to your kindness and selfless generosity.

**Reply by dpetka2001** [reply]
You just have to undo the changes that you made in those files and move those changes to your personal configuration. If you can't remember what changes you made then going into that directory which is a `.git` repo, doing `git diff` should show you the changes made. Or you can even delete that directory as it doesn't matter and will be recreated upon the next sync that will run.

Documentation has 2 sections regarding your configuration. Options, autocmds and keymaps which are described [here](https://www.lazyvim.org/configuration/general) and plugins [here](https://www.lazyvim.org/configuration/plugins).
